package.json

```diff
@@ -32,6 +32,9 @@
   "bugs": {
     "url": "https://github.com/cerner/terra-toolkit/issues"
   },
+  "bin": {
+    "terra": "./packages/terra-cli/bin/terra"
+  },
```

Comment:
Is this to pull forward the bin from the terra-ci package?

Reply:
This is something I saw lerna do in their top level.  But I don't think it actually does anything from what I can tell.  I can remove it.

Reply:
Removed: https://github.com/cerner/terra-toolkit/pull/460/commits/6c35f4191a2115b9652c22dc9f42105554d0b08c